cmd/helmify/flags.go

```diff
@@ -64,6 +64,7 @@ func ReadFlags() config.Config {
 	flag.BoolVar(&result.ImagePullSecrets, "image-pull-secrets", false, "Allows the user to use existing secrets as imagePullSecrets in values.yaml")
 	flag.BoolVar(&result.GenerateDefaults, "generate-defaults", false, "Allows the user to add empty placeholders for tipical customization options in values.yaml. Currently covers: topology constraints, node selectors, tolerances")
 	flag.BoolVar(&result.CertManagerAsSubchart, "cert-manager-as-subchart", false, "Allows the user to add cert-manager as a subchart")
+	flag.StringVar(&result.CertManagerVersion, "cert-manager-version", "v1.12.2", "Allows to define cert-manager version. Only useful with cert-manager-as-subchart.")
```

Comment:
previously, cert-manager version was omitted from the subchart meaning that the late version will be used. Now, a fixed version `1.12.2` will be used by default. Will it be better to omit the version if the flag is not set?

Reply:
got it, thanks!